src/semantize/typing.nit

```diff
@@ -327,49 +327,66 @@ private class TypeVisitor
 
 		assert mproperty isa MMethod
 
+		return get_method_by_property(node, recvtype, mproperty, recv_is_self)
+	end
+
+	fun get_method_by_property(node: ANode, recvtype: MType, mproperty: MMethod, recv_is_self: Bool): nullable CallSite
```

Comment:
Shouldn't it be `get_callsite...`

Also a bit of documentation can be useful here.

Reply:
The name was changed to `build_callsite_by ...` because the method allows you to create a new callsite (it will be more expressive than get_method).

A small documentation was also added.